src/gameplay_v2/roles/components/leader.ts

```diff
@@ -0,0 +1,8 @@
+import Component from './component';
+
+export class Leader extends Component {
+  name = 'leader';
+  data = {};
+}
```

Comment:
I don't think this should be a component. Instead, it should probably be in GameData in the game engine! This isn't a particular feature we want to compose other roles with!